src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -70,3 +70,13 @@ input.usa-only-phone {
     margin: 0;
   }
 }
+[type="submit"].vads-u-width--auto {
```

Comment:
this seems leftover from an older branch you were working on.

Reply:
Code is removed